sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/DatabaseAccountOperationsTests.cs

```diff
@@ -0,0 +1,257 @@
+            databaseAccountCreateOrUpdateParameters.IpRules.Add(new IpAddressOrRange("23.43.230.120"));
+            databaseAccountCreateOrUpdateParameters.IsVirtualNetworkFilterEnabled = true;
+            databaseAccountCreateOrUpdateParameters.EnableAutomaticFailover = false;
+            databaseAccountCreateOrUpdateParameters.ConnectorOffer = "Small";
+            databaseAccountCreateOrUpdateParameters.DisableKeyBasedMetadataWriteAccess = false;
+            DatabaseAccountGetResults databaseAccountGetResults1 =
+                await WaitForCompletionAsync(
+                    await CosmosDBManagementClient.DatabaseAccounts.StartCreateOrUpdateAsync(resourceGroupName, databaseAccountName, databaseAccountCreateOrUpdateParameters));
+            Assert.AreEqual(200, CosmosDBManagementClient.DatabaseAccounts.CheckNameExistsAsync(databaseAccountName).Result.Status);
+            DatabaseAccountGetResults databaseAccountGetResults2 = await CosmosDBManagementClient.DatabaseAccounts.GetAsync(resourceGroupName, databaseAccountName);
+            VerifyCosmosDBAccount(databaseAccountGetResults1, databaseAccountGetResults2);
+
+            var databaseAccountUpdateParameters = new DatabaseAccountUpdateParameters();
+            databaseAccountUpdateParameters.Tags.Add("key3", "value3");
+            databaseAccountUpdateParameters.Tags.Add("key4", "value4");
+            databaseAccountUpdateParameters.IsVirtualNetworkFilterEnabled = false;
+            databaseAccountUpdateParameters.EnableAutomaticFailover = true;
+            databaseAccountUpdateParameters.DisableKeyBasedMetadataWriteAccess = true;
+            await WaitForCompletionAsync(
+                await CosmosDBManagementClient.DatabaseAccounts.StartUpdateAsync(resourceGroupName, databaseAccountName, databaseAccountUpdateParameters));
+            var failoverPolicyList = new List<FailoverPolicy>();
+            failoverPolicyList.Add(new FailoverPolicy()
+            {
+                LocationName = "WEST US",
+                FailoverPriority = 0
+            });
+            FailoverPolicies failoverPolicies = new FailoverPolicies(failoverPolicyList);
+            await WaitForCompletionAsync(
+                await CosmosDBManagementClient.DatabaseAccounts.StartFailoverPriorityChangeAsync(resourceGroupName, databaseAccountName, failoverPolicies));
+            DatabaseAccountGetResults databaseAccountGetResults3 = await CosmosDBManagementClient.DatabaseAccounts.GetAsync(resourceGroupName, databaseAccountName);
+            VerifyCosmosDBAccount(databaseAccountGetResults3, databaseAccountUpdateParameters);
+            VerifyFailoverPolicies(failoverPolicyList, databaseAccountGetResults3.FailoverPolicies);
+
+            List<DatabaseAccountGetResults> databaseAccounts = await CosmosDBManagementClient.DatabaseAccounts.ListAsync().ToEnumerableAsync();
+            Assert.IsNotNull(databaseAccounts);
```

Comment:
There is too much going on in this test, you should consider splitting list into separate tests. There is no way to just test list for example without running this entire test before. One should be able to do dotnet test --filter TestList or something